### ISSUE TITLE
 Error was 'No such file or directory' and return code '256'

### ISSUE THREAD

**Issue by MAXINELSX** [HRGV/phyloFlash#169]
hi;
so sorry to bother you
my code is:
phyloFlash.pl -lib phyloFlash_db -almosteverything -read1 ST2019_10_8_FDME220033526-1r_HNJLMDSX2_L2_clean_1.fastq -read2 ST2019_10_8_FDME220033526-1r_HNJLMDSX2_L2_clean_2.fastq -readlength 150

and the error is 
[08:51:18] running subcommand:
	   /home/lishuxian/anaconda3/envs/pf/bin/bbmap.sh fast=t
	   minidentity=0.7 -Xmx20g reads=-1 threads=40 po=f
	   outputunmapped=f path=/home/lishuxian/138.1
	   out=phyloFlash_db.bbmap.sam
	   outm=phyloFlash_db.ST2019_10_8_FDME220033526-1r_HNJLMDSX2_L2_clean_1.fastq.SSU.1.fq
	   noheader=t ambiguous=all build=1
	   in=ST2019_10_8_FDME220033526-1r_HNJLMDSX2_L2_clean_1.fastq
	   bhist=phyloFlash_db.basecompositionhistogram
	   ihist=phyloFlash_db.inserthistogram
	   idhist=phyloFlash_db.idhistogram
	   scafstats=phyloFlash_db.hitstats overwrite=t
	   outm2=phyloFlash_db.ST2019_10_8_FDME220033526-1r_HNJLMDSX2_L2_clean_1.fastq.SSU.2.fq
	   pairlen=1200
	   in2=ST2019_10_8_FDME220033526-1r_HNJLMDSX2_L2_clean_2.fastq
	   2>phyloFlash_db.bbmap.out
**[09:01:19] FATAL: Tool execution failed!.
	   Error was 'No such file or directory' and return code '256'
	   Check error log file phyloFlash_db.bbmap.out
	   Aborting.**
[09:01:19] Saving log to file phyloFlash_log_on_error
I used this phyloFlash for over 150 samples, and only one sample showed this error, but my other data pretreatments and other analysis are the same and didn't show an error.
I checked the https://github.com/HRGV/phyloFlash/issues/95
Though the error code is the same, the reasons are not the same

**Comment by Xijiao000** [comment]
yes, the same error occurred in my work, how did you solve it? looking forward to your reply!

**Comment by Xijiao000** [comment]
I've solved this problem, mainly because the fastq file had errors in the compression and I recompressed it then it run fine.
